docs: release announcement for 1.22 (#164)
cloudnative-pg/cloudnative-pg.github.io: content/blog/cloudnative-pg-1-22.0-released/index.md

draft: false
image:
    url: wildlife-zoo-mammal-fauna-elephant-sri-lanka-988769-pxhere.com.jpg
    attribution: from <strong><a href="https://pxhere.com/en/photo/988769?utm_content=clipUser&utm_medium=referral&utm_source=pxhere">PxHere</a></strong>
author: gbartolini
tags:
 - release
 - postgresql
 - postgres
 - kubernetes
 - k8s
 - cloudnativepg
 - tablespaces
summary: The CloudNativePG community has released the new 1.22 minor version and a new update for the supported 1.21 and 1.20 versions of the CloudNativePG operator.
---
The **CloudNativePG Community** is thrilled to unveil version 1.22.0 of the
**CloudNativePG Operator**, a significant milestone featuring the introduction
of **declarative tablespaces** and **temporary tablespaces** alongside various
enhancements and fixes.

Tablespaces, a powerful and widely adopted feature in database management
systems, now take center stage in CloudNativePG 1.22.0. This release empowers
users to boost the vertical scalability of their databases by decoupling the
physical and logical data modeling, achieving optimal performance through
parallel on-disk read/write operations. With CloudNativePG, users can define

> additional tablespace volumes, and also ensure they are used to temporary

Reviewer: ```suggestion
additional tablespace volumes, and also ensure they can be used for temporary
```

- [Release notes for 1.22.0](https://cloudnative-pg.io/documentation/1.22/release_notes/v1.22/)
- [Release notes for 1.21.2](https://cloudnative-pg.io/documentation/1.21/release_notes/v1.21/)
- [Release notes for 1.20.5](https://cloudnative-pg.io/documentation/1.20/release_notes/v1.20/)

Thank you for your continued support and engagement with CloudNativePG!

The **CloudNativePG Community** has announced version 1.22.0, a new minor
release of the **CloudNativePG Operator**, which introduces **declarative
tablespaces** and **temporary tablespaces** along several enhancements and
fixes.

Tablespaces are a robust and widely embraced feature in database management
systems that enhances the vertical scalability of a database, by decoupling the
physical and logical modeling of data and by optimizing performance through
parallel on-disk read/write operations. CloudNativePG enables you to define
additional tablespace volumes, and control whether they can be used for
temporary operations (by seamlessly managing the `temp_tablespaces` GUC).

Tablespaces are another important step towards Very Large Database adoption,
following the introduction of Kubernetes volume snapshots in 1.21.

> From 1.22, the `ALTER SYSTEM` command will be disabled by default, to ensure

Reviewer: this and following ... it's a duplicate. @gbartolini are you experimenting different forms, or is this a copy-paste artifact?

Reviewer: Mostly dupe. I added a bit of this one to the "first copy".
Will approve this PR but not merge it or publish it, leaving that to you @gbartolini .
The release documents are now published.
